types/chrome/index.d.ts

```diff
@@ -1595,6 +1595,13 @@ declare namespace chrome.declarativeContent {
     /** Declarative event action that shows the extension's page action while the corresponding conditions are met. */
     export class ShowPageAction { }
 
+    /** Declarative event action that changes the icon of the page action while the corresponding conditions are met. */
+    export class SetIcon {
+        constructor (options: {
```

Comment:
```suggestion
        constructor (options?: {
```
Docs seems to says that's optional (`(optional) imageData`). Is that correct?